README.md

```diff
@@ -10,11 +10,11 @@ The Jaeger Operator is an implementation of a [Kubernetes Operator](https://kube
 To install the operator, run:
 ```
 kubectl create namespace observability
-kubectl create -f https://raw.githubusercontent.com/jaegertracing/jaeger-operator/master/deploy/crds/jaegertracing.io_jaegers_crd.yaml
-kubectl create -f https://raw.githubusercontent.com/jaegertracing/jaeger-operator/master/deploy/service_account.yaml
-kubectl create -f https://raw.githubusercontent.com/jaegertracing/jaeger-operator/master/deploy/role.yaml
-kubectl create -f https://raw.githubusercontent.com/jaegertracing/jaeger-operator/master/deploy/role_binding.yaml
-kubectl create -f https://raw.githubusercontent.com/jaegertracing/jaeger-operator/master/deploy/operator.yaml
+kubectl create -n observability -f https://raw.githubusercontent.com/jaegertracing/jaeger-operator/master/deploy/crds/jaegertracing.io_jaegers_crd.yaml
```

Comment:
This seems like a breaking change for installing the operator. 

Consider keeping the current approach working (install cluster-wide operator in observability namespace)

Reply:
Our options after this PR are:

1. We change the instructions to add two more `kubectl` commands, one for the `cluster_role.yaml` and one for the `cluster_role_binding.yaml`, with a note that this last one has to be manually changed if the namespace isn't `observability`. The default installation would then require cluster wide permissions.
1. We change the instructions so that people can install the operator on their own namespace no matter where they are deploying, as cluster wide permissions aren't required. This will make a couple of features off by default, but we are logging a message making them aware that the operator can work better with more permissions.

I have to say that I don't have a preference. One is the way we always did, the other is "secure by default".

So, here's the poll: @objectiser, @pavolloffay, @rubenvp8510, @kevinearls, @jkandasa, what's your preference?

Reply:
@annanay25 feel free to chime in as well

Reply:
@jpkrohling I don't have a strong preference, but if I had to choose it would be option 2.

Reply:
@jpkrohling I'd lean more towards the second option, but the docs should be clear how to get the additional permissions (so essentially be able to do option 1 themselves)?

Reply:
Agree with @kevinearls, option 2 sounds better.

Reply:
@jpkrohling Options 2 looks good to me.

Reply:
I also think option 2 seems to be the best option here.

Reply:
I just updated this PR to include a few extra lines on this.